cumulus-client/src/main/java/mil/army/usace/hec/cumulus/client/controllers/CumulusWatershedController.java

```diff
@@ -0,0 +1,71 @@
+package mil.army.usace.hec.cumulus.client.controllers;
+
+import static mil.army.usace.hec.cumulus.client.controllers.CumulusEndpointConstants.ACCEPT_HEADER_V1;
+
+import java.io.IOException;
+import java.util.List;
+import java.util.concurrent.CompletableFuture;
+import java.util.concurrent.CompletionException;
+import mil.army.usace.hec.cumulus.client.model.CumulusObjectMapper;
+import mil.army.usace.hec.cumulus.client.model.Watershed;
+import mil.army.usace.hec.cwms.http.client.ApiConnectionInfo;
+import mil.army.usace.hec.cwms.http.client.HttpRequestBuilderImpl;
+import mil.army.usace.hec.cwms.http.client.HttpRequestResponse;
+import mil.army.usace.hec.cwms.http.client.request.HttpRequestExecutor;
+
+public class CumulusWatershedController {
```

Comment:
Need to supply an ExecutorService for async calls.

Reply:
updated